test/unit/testExecutorOtherName.js

```diff
@@ -0,0 +1,46 @@
+/* eslint max-lines: ["error", 500] */
```

Comment:
Please add `'use strict;'` as the first line of this file. See https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#strict-mode for an example